offload/tools/offload-tblgen/MiscGen.cpp

```diff
@@ -61,7 +61,7 @@ void EmitOffloadImplFuncDecls(const RecordKeeper &Records, raw_ostream &OS) {
   OS << GenericHeader;
   for (auto *R : Records.getAllDerivedDefinitions("Function")) {
     FunctionRec F{R};
-    OS << formatv("{0}_impl_result_t {1}_impl(", PrefixLower, F.getName());
+    OS << formatv("llvm::Error {0}_impl(", F.getName());
```

Comment:
These declarations are used in the `llvm::offload` namespace, so you could just use `Error` here